spec/system/admin/flags_spec.rb

```diff
@@ -54,6 +54,12 @@
         expect(page).not_to have_content(submission_owner.username)
       end
     end
+
+    it 'shows the flag in the resolved list' do
+      visit '/admin/flags?scope=resolved'
```

Comment:
Nit pick, can we use a route helper here instead please?  `visit admin_flags_path(scope: 'resolved')` that will throw up a nice error if these urls ever change.

Reply:
done :+1: